checkov/terraform/checks/resource/aws/DynamoDBTablesEncrypted.py

```diff
@@ -15,10 +15,11 @@ def __init__(self):
     def scan_resource_conf(self, conf: Dict[str, List[Any]]) -> CheckResult:
         if 'server_side_encryption' in conf.keys():
             sse = conf['server_side_encryption'][0]
-            enabled = sse.get("enabled")
-            kms_key_arn = sse.get("kms_key_arn")
-            if enabled == [True] and kms_key_arn is not None:
-                return CheckResult.PASSED
+            if isinstance(sse, Dict):
```

Comment:
I'm actually surprised it works like this
```suggestion
            if isinstance(sse, dict):
```